lib/services/build.ts

```diff
@@ -420,7 +421,7 @@ export class BuildService implements Project.IBuildService {
 
 			this.$loginManager.ensureLoggedIn().wait();
 
-			if(this.$mobileHelper.isWP8Platform(platform) && this.$project.projectData.WPSdk && this.$project.projectData.WPSdk === "8.0" && helpers.versionCompare(this.$project.projectData.FrameworkVersion,"3.7.0") >= 0) {
+			if(this.$project.projectData.Framework === this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova && this.$mobileHelper.isWP8Platform(platform) && this.$project.projectData.WPSdk && this.$project.projectData.WPSdk === "8.0" && semver.gte(this.$project.projectData.FrameworkVersion,"3.7.0")) {
```

Comment:
This is a little bit strange to be here. Consider moving it to `cordova-project`.